audio/music/Youtube_Audio_Library.md

```diff
@@ -0,0 +1 @@
+You can use the YouTube Audio Library to get free music and sound effects to use in your videos. Learn more about how to move viewers in your videos with sound. You can download and use some of our high-quality 320kbps audio tracks and sound effects royalty-free.
```

Comment:
Please follow the documentation about adding a music and 

- [ ] create a license file, this is not a license file but a vision
- [ ] find out the license(s) of the video and copy the whole text into the file

Reply:
@niccokunzmann This is a common license file for all the Audios provided by Youtube Audio Library. This has no Copyright Issues https://www.youtube.com/audiolibrary/music

Reply:
@naveentherocker Idk about this:
> YouTube cannot give you information about legal liability or other issues that can occur off the platform. You may wish to consult a qualified attorney if you have questions about your use of music.

https://support.google.com/youtube/answer/3376882?hl=en

But we can't get it's license:
https://productforums.google.com/forum/#!topic/youtube/w6Wm4AqYxCs